tests/acceptance/save-map-button-map-type-editing-view-test.js

```diff
@@ -0,0 +1,67 @@
+import { module, test } from 'qunit';
+import {
+  visit,
+  click,
+  fillIn,
+  currentURL,
+} from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+import mapboxGlLoaded from '../helpers/mapbox-gl-loaded';
+import mapboxGlDrawReady from '../helpers/mapbox-gl-draw-ready';
+import plotVerticies from '../helpers/plot-verticies';
+
+module('Acceptance | user can click save map button from edit screen', function(hooks) {
```

Comment:
This test seems like a duplicate but maybe this is due to #158 not being merged yet